http/README.md

```diff
@@ -0,0 +1,168 @@
+# HTTP(S) Listener Source
+
+A Hazelcast Jet source for listening HTTP(S) requests, and a sink which
+clients can connect and listen the items using either websocket or
+server-sent events.
+
+The source creates an [Undertow](https://github.com/undertow-io/undertow)
+server for each processor and start listening the incoming requests.
+User can configure the host and port for the server. User can also
+define a ssl context to secure the connections. The payload can be
+converted to the desired output before emitting to the downstream.
+
+The sink is not distributed. Sink creates a single [Undertow](https://github.com/undertow-io/undertow)
```

Comment:
We could have a distributed sink - if 
- we do grouping by a key
- the client is interested only for events for particular key, so it connects to that particular server 

Just an idea, no need to do anything now.